sql-streaming-mqtt/src/main/resources/META-INF/services/org.apache.spark.sql.sources.DataSourceRegister

```diff
@@ -0,0 +1 @@
+org.apache.bahir.sql.streaming.mqtt.MQTTStreamSourceProvider
```

Comment:
AFAIK, This particular file does not allow for comments. See [here](https://github.com/apache/spark/blob/master/sql/core/src/main/resources/META-INF/services/org.apache.spark.sql.sources.DataSourceRegister)